spec/cell_spec.rb

```diff
@@ -46,12 +46,10 @@
       expect(@cell_1.render).to eq(".")
       @cell_1.fire_upon
       expect(@cell_1.render).to eq("M")
-
+      
       @cell_2.place_ship(@cruiser)
       expect(@cell_2.render).to eq(".")
-    end
 # show the ship
-    it 'it can render with optional argument' do
```

Comment:
We had a bit of struggle here until we realized that removing the end on line 52 allowed our test to pass

Reply:
As well as line 52